app/helpers/schools/dashboard_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Schools
+  module DashboardHelper
+    def manage_ects_and_mentors?(school_cohort)
+      !(school_cohort.school.cip_only? || school_cohort.school_chose_diy?)
```

Comment:
```suggestion
      school_cohort.full_induction_programme? || school_cohort.core_induction_programme?
```

Reply:
This would make more sense to me but may not fix the UI for all of the ones that currently have selected the wrong programme despite until the underlying data is fixed (if necessary?)